lib/qldb.connect.js

```diff
@@ -10,7 +10,11 @@ class qldbConnect {
      * @param serviceConfigOptions
      */
     constructor(ledgerName, serviceConfigOptions) {
-        this.qldbDriver = new PooledQldbDriver(ledgerName, serviceConfigOptions);
+        const {
+            retryLimit, poolSize, timeoutMillis, ...options
+        } = serviceConfigOptions;
+
+        this.qldbDriver = new PooledQldbDriver(ledgerName, options, retryLimit, poolSize, timeoutMillis);
```

Comment:
In the final 1.0.0 release of the driver the PooledQldbDriver is depricated. It wil be replaced by 
this.qldbDriver = new QldbDriver(ledgerName, serviceConfigOptions);
My master branch already has this, maybe you could merge this in yours.